static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap

```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
can you add comments in the test file on how to update these snap files?

Reply:
done.